src/ansible_navigator/actions/_actions.py

```diff
@@ -52,54 +59,95 @@ def _import_all(package: str) -> None:
 
 
 def register(cls: Any) -> Any:
-    """Decorator for registering a new action"""
+    """Register an action, used as a decorator.
+
+    :param cls: The class to register
+    :returns: The class after registration
+    """
     package, _, action = cls.__module__.rpartition(".")
     pkg_info = _ACTIONS.setdefault(package, {})
     pkg_info[action] = ActionT(name=action, cls=cls, kegex=re.compile(cls.KEGEX))
     return cls
 
 
 def get(package: str, action: str) -> Callable:
-    """Get a given action"""
+    """Import and return a given action.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :returns: The action's registered class
+    """
     _import(package, action)
     return _ACTIONS[package][action].cls
 
 
 def get_factory(package: str) -> Callable:
-    """Create a get() function for one package"""
+    """Create a ``get`` function for one package.
+
+    :param package: The name of the package
+    :returns: The action's registered class
+    """
     return functools.partial(get, package)
 
 
 def kegex(package: str, action: str) -> Tuple:
-    """Return a tuple of name, kegex for a action"""
+    """Return a tuple of name, class, kegex for a action.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :returns: The name, class and kegex for an action
+    """
     _import(package, action)
     return _ACTIONS[package][action]
 
 
 def kegexes(package: str) -> Generator:
-    """Return a tuple of tuples, name, kegex"""
+    """Return a tuple of tuples, name, kegex for all actions.
+
+    :param package: The name of the package
+    :returns: A generator for all kegexs
+    """
     _import_all(package)
     return (kegex(package, name) for name in names(package))
 
 
 def kegexes_factory(package: str) -> Callable:
-    """Create a kegexs() function for all packages"""
+    """Create a ``kegexs`` function for all packages.
+
+    :param package: The name of the package
+    :returns: A ``kegexes`` method for the package
+    """
     return functools.partial(kegexes, package)
 
 
 def names(package: str) -> List:
-    """List all actions in one package"""
+    """List all actions in one package.
+
+    :param package: The name of the package
+    :returns: All packages
+    """
     _import_all(package)
     return sorted(_ACTIONS[package])
 
 
 def names_factory(package: str) -> Callable:
-    """Create a names() function for one package"""
+    """Create a ``names`` function for one package.
+
+    :param package: The name of the package
+    :returns: a ``names`` method for the package
```

Comment:
I suppose this could link either https://docs.python.org/3/library/collections.abc.html#collections.abc.Callable or a glossary term explanation like it's done here https://docs.aiohttp.org/en/stable/glossary.html#term-callable.
```suggestion
    :returns: a ``names`` :class:`callable <collections.abc.Callable>` for the package
```

Reply:
nice